lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -39,27 +42,35 @@ Stream<Patch> intlImporter(
   if (!needsIntlImport) return;
 
   final intlUri = 'package:${projectName}/src/intl/${projectName}_intl.dart';
-
+  final currentDirectory = Directory.current;
+  final intlDirectory = path.join(currentDirectory.path, 'lib/src/intl/${projectName}_intl.dart');
+  final relativePathToIntlDir = path.relative(intlDirectory, from: path.current);
```

Comment:
Got it I have updated the and direct use this `Directory.current.path` instead of taking in variable 
  `final intlDirectory = path.join(Directory.current.path, intlFilePath);`